UI/Selenium/Hooks/Hooks.cs

```diff
@@ -46,24 +51,33 @@ public static void BeforeTestRun()
                 var reporter = new ExtentHtmlReporter(PathReport);
                 _extent = new ExtentReports();
                 _extent.AttachReporter(reporter);
-                //log
-                logger.Info(" Automation Testing Execution Commenced");
+                Logger.InfoWithDate("Automation Test Execution Commenced");
             }
-            catch (Exception e)
+            catch (Exception ex)
             {
-                logger.Error(" Exception has occurred " + e);
+                Logger.Error(ex,"An error has occured");
```

Comment:
Error needs a date time like Info to be consistent.

Exception needs detail, I'd expect to see the message & Stacktrace, and hopefully more such as what caused the exception.

Reply:
Error now includes Stacktrace and a timestamp, just like Info